src/accel/LocalTransporter.cc

```diff
@@ -262,8 +262,7 @@ auto LocalTransporter::GetActionTime() const -> MapStrReal
         CELER_ASSERT(action_ptrs.size() == time.size());
         for (auto i : range(action_ptrs.size()))
         {
-            auto&& label = action_ptrs[i]->label();
-            result[label] = time[i];
+            result[std::string{action_ptrs[i]->label()}] = time[i];
```

Comment:
I don't think we can do it for `TimerOutput` because that might exceed the lifetime of the core params... I figured it was safer to do a few one-time string copies than to risk it...

Reply:
So far we only use string literals that have static storage, but sure in general it's less error-prone for the map to own the key. It just doesn't look nice having to instantiate a string to look up the map when the string_view has the information needed. Gotta wait for C++20 support to fix that.

Reply:
Do you mean C++26? (which changes `operator[]`)?
Or should we use [`unordered_map::find`](https://en.cppreference.com/w/cpp/container/unordered_map/find) which is already (in C++14) templated on the parameter type?

Reply:
I think you did mean C++20 as described here: https://www.cppstories.com/2021/heterogeneous-access-cpp20/.   I.e. the templated function/operator is not enough, one also has to customize the hash comparisons.

Reply:
Bummer, I saw that and hoped it would be automatic for the string/view case